src/jit/codegencommon.cpp

```diff
@@ -10162,11 +10162,14 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 #endif // DEBUG
 
         assert(PSP_slot_CallerSP_offset < 0);
-        assert(compiler->lvaPSPSym != BAD_VAR_NUM);
-        assert(PSP_slot_CallerSP_offset == compiler->lvaGetCallerSPRelativeOffset(compiler->lvaPSPSym)); // same offset
-                                                                                                         // used in main
-                                                                                                         // function and
-                                                                                                         // funclet!
+        if (!compiler->IsTargetAbi(CORINFO_CORERT_ABI)) // PSPSym is not used by the CoreRT ABI
```

Comment:
lvaPSPSym is initialized with a BAD_VAR_NUM in lclvars.cpp+76 and doesn't change because in lclvars.cpp+3871:
```
// PSPSym and LocAllocSPvar are not used by the CoreRT ABI                                                                                                                               
if (!IsTargetAbi(CORINFO_CORERT_ABI))
{
#if FEATURE_EH_FUNCLETS
    if (ehNeedsPSPSym())
    {
        lvaPSPSym            = lvaGrabTempWithImplicitUse(false DEBUGARG("PSPSym"));
        LclVarDsc* lclPSPSym = &lvaTable[lvaPSPSym];
        lclPSPSym->lvType    = TYP_I_IMPL;
    }
#endif // FEATURE_EH_FUNCLETS
```